node/src/gql/query/me/upload-tasks.ts

```diff
@@ -31,8 +31,13 @@ export const uploadTasks = {
       throw new Error('you do not have a mentor');
     }
     if (args.mentorId && `${mentor._id}` !== `${args.mentorId}`) {
-      if (context.user.userRole !== UserRole.ADMIN) {
-        throw new Error('you do not have permission to edit this mentor');
+      if (
+        context.user.userRole !== UserRole.ADMIN &&
+        context.user.userRole !== UserRole.CONTENT_MANAGER
+      ) {
+        throw new Error(
+          'you do not have permission to view this mentors information'
+        );
       }
       mentor = await MentorModel.findById(args.mentorId);
```

Comment:
I think mentorId is under context instead of args for some reason?
```
    args: any,
    context: { user: User; mentorId: string }
```
Should be under args

Reply:
a47273f